client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -267,13 +266,17 @@ class EmailPlan extends React.Component {
 	}
 
 	renderAddNewMailboxesOrRenewNavItem() {
-		const { domain, hasSubscription, purchase, selectedSite, translate } = this.props;
+		const { domain, purchase, translate } = this.props;
 
-		if ( ! domain.currentUserCanManage || ! hasSubscription ) {
-			return null;
+		if ( hasEmailForwards( domain ) ) {
+			return (
+				<VerticalNavItem { ...this.getAddMailboxProps() }>
+					{ translate( 'Add new email forwards' ) }
+				</VerticalNavItem>
+			);
 		}
 
-		if ( ! selectedSite || ! purchase ) {
+		if ( ! purchase ) {
```

Comment:
This logic still doesn't feel right -- we won't have a `purchase` for all MVP subscriptions, so we should not show a purchase for that case. I think the logic should be adjusted slightly:
```
if ( hasSubscription ) { // assuming this can be resolved immediately and is not dependent on async data loading
  if ( ! purchase ) {
    // return placeholder
  }

  if ( isExpired( purchase ) ) {
    // return renewal message
  }
}

// return add new mailboxes content
```